src/components/30-organisms/commercial-hero-banner/index.scss

```diff
@@ -0,0 +1,83 @@
+.o-commercial-hero-banner {
+  @include breakpoint($mediaquery-lg-up) {
+    min-height: 560px;
```

Comment:
You could define a variable for the height, because you are using it several times.

Reply:
Right, this makes sense now.